components/searchable_user_list/searchable_user_list_container.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {UserProfile} from 'mattermost-redux/types/users';
+import {Channel, ChannelMembership} from 'mattermost-redux/types/channels';
+import {TeamMembership} from 'mattermost-redux/types/teams';
+
+import SearchableUserList from './searchable_user_list';
+
+type Props = {
+    users: UserProfile[] | null;
+    usersPerPage: number;
+    total: number;
+    extraInfo?: {[key: string]: Array<string | JSX.Element>};
+    nextPage: (page: number) => void;
+    search: (term: string) => void;
+    actions?: React.ReactNode[];
+    actionProps?: {
+        mfaEnabled: boolean;
+        enableUserAccessTokens: boolean;
+        experimentalEnableAuthenticationTransfer: boolean;
+        doPasswordReset: (user: UserProfile) => void;
+        doEmailReset: (user: UserProfile) => void;
+        doManageTeams: (user: UserProfile) => void;
+        doManageRoles: (user: UserProfile) => void;
+        doManageTokens: (user: UserProfile) => void;
+        isDisabled: boolean | undefined;
+    };
+    actionUserProps?: {
+        [userId: string]: {
+            channel?: Channel;
+            teamMember: TeamMembership;
+            channelMember?: ChannelMembership;
+        };
+    };
+    focusOnMount?: boolean;
+}
+
+type State = {
+    term: string;
+    page: number;
+};
+
+export default class SearchableUserListContainer extends React.PureComponent<Props, State> {
```

Comment:
I wonder if we could turn this into a functional component? We are trying to use functional components for smaller components

Reply:
Done. Good to know, thanks. I will keep in my in future PRs